spec/factories/services.rb

```diff
@@ -4,6 +4,7 @@
     association :service_type
     date_started Date.new(2014, 9, 9)
     recorded_at Date.new(2014, 11, 2)
+    discontinued_at Date.new(2014, 12, 2)
```

Comment:
adding this to services factory threw the tests off a little bit which I adjusted below.